tests/phpunit/tests/term/query.php

```diff
@@ -867,4 +867,114 @@ public function test_hide_empty_should_include_empty_parents_of_nonempty_childre
 
 		$this->assertContains( $t1, $q->terms );
 	}
+
+	/**
+	 * Ensure cache keys are generated without WPDB placeholders.
+	 *
+	 * @ticket 57298
+	 *
+	 * @covers       WP_Term_Query::generate_cache_key
+	 * @dataProvider data_query_cache
+	 */
+	public function test_generate_cache_key_placeholder( $args ) {
+		global $wpdb;
+		$query1 = new WP_Term_Query();
+		$query1->query( $args );
+
+		$query_vars = $query1->query_vars;
+		$request    = $query1->request;
+
+		$reflection = new ReflectionMethod( $query1, 'generate_cache_key' );
+		$reflection->setAccessible( true );
+
+		$cache_key_1 = $reflection->invoke( $query1, $query_vars, $request );
+
+		$request_with_placeholder = $wpdb->remove_placeholder_escape( $request );
```

Comment:
Should this be `_without_`? My reading of `remove_placeholder_escape` is that it replaces the placeholder with the `%` symbol

Reply:
Fixed